api/openapi/v2/core-metadata.yaml

```diff
@@ -2395,41 +1936,9 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ConfigResponse'
-        '207':
-          description: "Indicates a multi-part response supportive of accepting multiple requests at once. The 'statusCode' property of each response in the returned array will indicate success or failure."
-          headers:
-            X-Correlation-ID:
-              $ref: '#/components/headers/correlatedResponseHeader'
-          content:
-            application/json:
-              schema:
-                type: array
-                items:
-                  anyOf:
-                    - $ref: '#/components/schemas/ErrorResponse'
-                    - $ref: '#/components/schemas/ConfigResponse'
-        '400':
```

Comment:
We will need an error response if we remove the BaseResponse. Use `500` and description of `Interval Server Error`

if we stay with using BaseResponse, then only code is the `207` with the specific code in the BaseResponse.

Reply:
Added 500 status code to config, ping, metric, version API.